src/test/java/teammates/test/cases/ui/StudentCourseDetailsPageActionTest.java

```diff
@@ -112,8 +117,66 @@ public void testExecuteAndPostProcess() {
         
     }
 
+    @Test
+    public void testTeamMemberDetailsOnViewTeamPage() throws EntityDoesNotExistException {
+        AccountAttributes student = dataBundle.accounts.get("student1InCourse1");
+        
+        String[] submissionParams = createValidParamsForProfile();
+        StudentProfileAttributes expectedProfile = getProfileAttributesFrom(submissionParams);
+        gaeSimulation.loginAsStudent(student.googleId);
+        
+        // adding profile picture for student1InCourse1
+        StudentProfileEditSaveAction action = getStudentProfileEditSaveAction(submissionParams);
```

Comment:
This is not adding profile picture; this is adding student profile which, while may be good, is not our intention here.